packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/simulation/SimulationUpdate.java

```diff
@@ -0,0 +1,43 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.simulation;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.PrePersist;
+import jakarta.validation.constraints.NotNull;
+import java.sql.Timestamp;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.UUID;
+import lombok.Data;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.models.dataservice.JsonConverter;
+
+@Data
+@TSModel
+@Entity
+public class SimulationUpdate {
+
+	@Id
+	@NotNull private UUID id = UUID.randomUUID();
+
+	@NotNull private UUID simulationId;
```

Comment:
How is this working without a reference back to the owning project for hibernate? I would have expected it to look like:

```
@ManyToOne
	@JoinColumn(name = "simulation_id")
	@JsonBackReference
	private SImulation simulation;
```

Reply:
I tried to do it as a unidirectional relationship, because I dont really care for it to link back. But that has hibernate treat it as a many to many... oging to make it bidirectional. 

I hate hibernate